$OpenBSD: patch-libavutil_x86_float_dsp_init_c,v 1.1 2015/05/01 08:41:30 ajacoutot Exp $

Disable AVX / FMA3 float SIMD functions for now which are currently
crashing when using VLC with AAC.

--- libavutil/x86/float_dsp_init.c.orig	Wed Jun 18 01:20:32 2014
+++ libavutil/x86/float_dsp_init.c	Fri May  1 04:18:40 2015
@@ -86,14 +86,20 @@ av_cold void ff_float_dsp_init_x86(AVFloatDSPContext *
         fdsp->vector_dmul_scalar = ff_vector_dmul_scalar_sse2;
     }
     if (EXTERNAL_AVX(cpu_flags)) {
+#if 0
         fdsp->vector_fmul = ff_vector_fmul_avx;
+#endif
         fdsp->vector_fmac_scalar = ff_vector_fmac_scalar_avx;
         fdsp->vector_dmul_scalar = ff_vector_dmul_scalar_avx;
+#if 0
         fdsp->vector_fmul_add    = ff_vector_fmul_add_avx;
+#endif
         fdsp->vector_fmul_reverse = ff_vector_fmul_reverse_avx;
     }
     if (EXTERNAL_FMA3(cpu_flags)) {
         fdsp->vector_fmac_scalar = ff_vector_fmac_scalar_fma3;
+#if 0
         fdsp->vector_fmul_add    = ff_vector_fmul_add_fma3;
+#endif
     }
 }
